packages/orchestration/src/types.d.ts

```diff
@@ -3,10 +3,6 @@ import type { Timestamp } from '@agoric/time';
 import type { Invitation } from '@agoric/zoe/exported.js';
 import type { Any } from '@agoric/cosmic-proto/google/protobuf/any';
 import type { AnyJson } from '@agoric/cosmic-proto';
-import type {
-  MsgBeginRedelegateResponse,
-  MsgUndelegateResponse,
-} from '@agoric/cosmic-proto/cosmos/staking/v1beta1/tx.js';
 import type {
   Delegation,
   Redelegation,
```

Comment:
should `Delegation`, `Redelegation`, and `UnbondingDelegation` come from cosmos-sdk like this?

It looks like this change happened Apr 24 when the Apr 18 spec was copied from the agoric-labs/orchestration-api-spec repo.

https://github.com/agoric-labs/orchestration-api-spec/blob/8775468/src/delegation.d.ts
https://github.com/agoric-labs/orchestration-api-spec/blob/00669e6/src/orchestration.d.ts

https://github.com/Agoric/agoric-sdk/blob/1e054ac972ddfff5fb03738747c69eecd125c463/packages/orchestration/src/types.d.ts

Reply:
for follow-up, see #9356